diff -upr dapl-2.0.34/dat/common/dat_strerror.c dapl-2.0.34.tmp/dat/common/dat_strerror.c
--- dapl-2.0.34/dat/common/dat_strerror.c	2011-08-19 13:09:23.000000000 -0400
+++ dapl-2.0.34.tmp/dat/common/dat_strerror.c	2013-11-12 16:17:04.000000000 -0500
@@ -601,7 +601,7 @@ dat_strerror(IN DAT_RETURN value,
 
 	if (DAT_SUCCESS != dat_strerror_major(value, major_message)) {
 		return DAT_INVALID_PARAMETER;
-	} else if (minor_message != NULL) {
+	} else if (minor_message) {
 		if (DAT_SUCCESS != dat_strerror_minor(value, minor_message)) {
 			return DAT_INVALID_PARAMETER;
 		}
